config/initializers/aws.rb

```diff
@@ -0,0 +1,9 @@
+require 'aws-sdk'
+require 'aws-sdk-resources'
+
+AWS.config(
+  :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.